src/card.css

```diff
@@ -0,0 +1,114 @@
+.card {
+  background: var(--clear_white-color);
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card--position {
+  display: flex;
+  flex-direction: column;
+  padding: 0 16px;
+  max-width: 200px;
+}
+
+.card__logo {
+  align-self: center;
+  height: 134px;
+  width: 160px;
+  margin: 32px 0 40px;
+}
+
+.card__description {
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  margin-bottom: 4px;
+  color: var(--dark_blue-for-text);
+}
+
+.card__code {
+  margin-bottom: 16px ;
+  color: var(--gray-for-text);
+}
+
+.card__raiting {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 24px;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(./images/star.svg);
+  background-position: 50%;
+  background-repeat: no-repeat;
+  height: 16px;
+  width: 16px;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars__star:nth-child(n):not(.stars__star:nth-child(5)) {
+  background-image: url(./images/star-active.svg);
+}
+
+.card__rating-number {
+  color: var(--dark_blue-for-text);
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 16px;
+}
+
+.card__price-text {
+  font-size: 12px;
+  font-weight: 400px;
+  line-height: 18px;
+  color: var(--gray-for-text) ;
+}
+
+.card__price-amount {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  color: var(--dark_blue-for-text);
+}
+
+.card__button {
+  display: inline-flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.card__button-buy {
+  text-align: center;
+  background-color: var(--blue-color);
+  border-radius: 5px;
+  color: var(--clear_white-color);
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 40px;
+  text-transform: uppercase;
+  text-decoration: none;
+  margin-bottom: 16px;
+  width: 166px;
+  height: 40px;
+  border: 1px solid transparent;
+}
+
+.card__button-buy:hover {
+  background-color: var(--clear_white-color);
+  color: var(--blue-color);
+  border: 1px solid var(--blue-color);
```

Comment:
In `.card__button-buy` class I would add color and remove the border from `hover`

Reply:
yes fixed